lib/Doctrine/Migrations/Tools/Console/Helper/MigrationDirectoryHelper.php

```diff
@@ -15,6 +15,9 @@
 use function rtrim;
 use function sprintf;
 
+/**
+ * @internal
```

Comment:
console helpers should not be internal either, as they must be registered in the CLI Application too (`cli-config.php` is expected to instantiate a HelperSet with the appropriate helpers in it)

Reply:
To be honest I am not sure why this is a symfony console helper. It should not be. It is only used directly here as an internal dependency https://github.com/doctrine/migrations/blob/31051801be855a45a1d34dcad3a92b37abebda24/lib/Doctrine/Migrations/Generator/Generator.php#L91

Reply:
well, this one should not be a console helper then. But other console helpers should not be internal.